pkg/command/utils/util.go

```diff
@@ -72,3 +72,18 @@ func GenerateJSONFile(jsonData []byte, targetJSON string) error {
 	}
 	return nil
 }
+
+func CheckOutputLocation(outputLocation string) (string, error) {
+	if outputLocation[len(outputLocation)-1:] == "/" {
```

Comment:
Ditto as the above comment, no need to trim the trailing `/` and return it as a string if you are using `filepath.Join`

Reply:
Changed the code as you suggested.